File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/trash/Signature.java
package gov.va.med.mhv.sm.model;

import java.io.Serializable;



public class Signature extends BaseModel implements Serializable{

	/**
	 * 
	 */
	private static final long serialVersionUID = 6814862738195064844L;
	protected String name;
	protected User user;
	protected String title;

	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public User getUser() {
		return user;
	}
	public void setUser(User user) {
		this.user = user;
	}
	public String getTitle() {
		return title;
	}
	public void setTitle(String title) {
		this.title = title;
	}
	
	public boolean equals(Object a){
		try{
			Signature x = (Signature)a;
			return this.getId().equals(x.getId()); 
		}catch(Exception e){
			return false;
		}
	}
	
	public int hashCode(){
		return this.getId().hashCode();
	}
		
}

File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/Signature.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace gov.va.medora.mdo.domain.sm
{
    [Serializable]
    public class Signature : BaseModel
    {
        private string _name;

        protected string Name
        {
            get { return _name; }
            set { _name = value; }
        }
        private gov.va.medora.mdo.domain.sm.User _user;

        protected gov.va.medora.mdo.domain.sm.User User
        {
            get { return _user; }
            set { _user = value; }
        }
        private string _title;

        protected string Title
        {
            get { return _title; }
            set { _title = value; }
        }
    }
}
